src/m365/pp/commands/gateway/gateway-get.spec.ts

```diff
@@ -0,0 +1,135 @@
+import * as assert from "assert";
+import * as sinon from "sinon";
+import appInsights from "../../../../appInsights";
+import auth from "../../../../Auth";
+import { Cli } from "../../../../cli/Cli";
+import { CommandInfo } from "../../../../cli/CommandInfo";
+import { Logger } from "../../../../cli/Logger";
+import Command, { CommandError } from "../../../../Command";
+import request from "../../../../request";
+import { sinonUtil } from "../../../../utils/sinonUtil";
+import commands from "../../commands";
+const command: Command = require("./gateway-get");
+
+describe(commands.GATEWAY_GET, () => {
+  let log: string[];
+  let logger: Logger;
+  let loggerLogSpy: sinon.SinonSpy;
+  let commandInfo: CommandInfo;
+
+  const gateway: any = {
+    id: "1f69e798-5852-4fdd-ab01-33bb14b6e934",
+    name: "My_Sample_Gateway",
+    type: "Resource",
+    publicKey: {
+      exponent: "AQAB",
+      modulus: "o6j2....cLk="
+    }
+  };
+
+  before(() => {
+    sinon.stub(auth, "restoreAuth").callsFake(() => Promise.resolve());
+    sinon.stub(appInsights, "trackEvent").callsFake(() => {});
+
```

Comment:
Added our stub for retrieving the process name from pid. Related to https://github.com/pnp/cli-microsoft365/pull/3769